dynamiqs/plots/plots_wigner.py

```diff
@@ -224,3 +230,64 @@ def plot_wigner_mosaic(
             clear=False,
         )
         ax.set(xlabel='', ylabel='', xticks=[], yticks=[])
+
+
+def plot_wigner_gif(
+    states: ArrayLike,
+    length_s: int = 5,
```

Comment:
What is this? Could we find a more explicit name in the absence of documentation?

Reply:
It plots a wigner in a gif. I will write a doc

Reply:
How about `gif_duration` then?

Reply:
And it should be a float I guess.